apps/LineRider/install

```diff
@@ -1,19 +1,35 @@
 #!/bin/bash
 
-version=2022.5.3
 #Bot automatically updates this when I make a new release.
+version=2022.5.3
+#Buster version
+version2=2022.5.1
 
+status "Installing Depends... "
 install_packages apt-transport-https dirmngr gnupg ffmpeg mono-devel unzip zip || exit 1
+status_green "Done."
 
 mkdir -p ~/LineRider
 cd ~/LineRider || error "Could not move to folder"
 
+if [[ $(get_codename) == "buster" ]]; then
+  use_version="$version2"
+elif [[ $(get_codename) == "bullseye" ]]; then
+  use_version="$version"
+else
+  warning "OS unknown, defaulting to Raspberry PI OS Bullseye install (If you do not have Raspberry Pi OS, ignore this.)"
+  use_version="$version"
+fi
+
```

Comment:
I think it would be better to not display a warning if the system is neither buster nor bullseye.
Do `if buster;then, else`

Reply:
Like that?